colossus-metrics/src/main/scala/colossus/metrics/TestKit.scala

```diff
@@ -0,0 +1,16 @@
+package colossus.metrics
+package testkit
+
+import akka.testkit.TestProbe
+import scala.reflect.ClassTag
+
+
+/**
+ * A Fake shared collection for creating event collectors using a provided test probe
+ */
+class TestSharedCollection(probe: TestProbe) extends Collection[SharedLocality] {
```

Comment:
Right now there actually is no class or object, just a method on `MetricSystem`, so I'll leave it be for now